### PR TITLE
Inline related component

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
screen shot 2015-03-12 at 14 56 05

### CODE REVIEW

**Comment by matthew-andrews** [comment]
i've fixed the build failure ( i think ) so hold off on merging this and once that's merged we can rebase/merge from that into this then merge this?

File: server/controllers/capi.js [hunk 1]
```diff
@@ -78,6 +78,12 @@ module.exports = function(req, res, next) {
 					$('blockquote').attr('class', 'article__block-quote o-quote o-quote--standard');
 					$('pull-quote').replaceWith(pullQuotesTransform);
 
+					// insert test related
+					if (articleV1._source.item.package.length > 0 && $('ft-paragraph').length >= 6) {
```
Comment (matthew-andrews) [inline server/controllers/capi.js]: hmm…  i thought `_source` was elastic search only?

Comment (matthew-andrews) [inline server/controllers/capi.js]: we should support elastic search on and off (try toggling the flag)

Comment (ironsidevsquincy) [inline server/controllers/capi.js]: absolutely, silly mistake. another reason to move into the `fetchCapiV1`

**Pull request merged**